packages/Lists/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@slup/lists",
+  "version": "0.0.1",
+  "description": "Material design lists",
+  "main": "dist.js",
+  "author": "Gejsi",
+  "license": "MIT",
+  "devDependencies": {
+    "@slup/ripple": "^0.0.2",
+    "babel-core": "6.25.0",
+    "babel-loader": "7.1.1",
+    "babel-plugin-inferno": "3.2.0",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-plugin-transform-decorators-legacy": "1.3.4",
+    "babel-plugin-transform-object-rest-spread": "6.23.0",
+    "babel-preset-es2015": "6.24.1",
+    "babili-webpack-plugin": "0.1.2",
+    "cross-env": "5.0.4",
+    "decko": "1.2.0",
+    "inferno": "3.7.1",
+    "inferno-component": "3.7.1",
+    "webpack": "3.4.1"
+  },
+  "scripts": {
+    "compile:watch": "webpack --watch --progress",
+    "compile:build": "cross-env NODE_ENV=production webpack --progress",
+    "prepublish": "yarn run compile:build"
+  },
+  "dependencies": {
+    "inferno-compat": "^3.7.1",
```

Comment:
Lock these and put them into devDependencies.
Locking means removing the ^ from the version.

Remeber to remove the yarn.lock and generate a new one once they are locked.